packages/commands/src/route-tokens-to-user-bank.mjs

```diff
@@ -0,0 +1,249 @@
+  )
+  .addOption(
+    new Option('-m, --mint [mint]', 'The currency to use')
+      .choices(['audio', 'usdc'])
+      .default('usdc')
+  )
+  .option('-f, --from <from>', 'The account to send tokens to (handle)')
+  .option('--memo <memo>', 'A data string to attach as a memo')
+  .action(async (amountInput, { from, mint, memo }) => {
+    const amount = BigInt(amountInput)
+    const audiusLibs = await initializeAudiusLibs(from)
+    const { solanaWeb3Manager } = audiusLibs
+
+    const { userbank: userbankPublicKey } =
+      await solanaWeb3Manager.createUserBankIfNeeded({
+        mint
+      })
+
+    if (!process.env.SOLANA_ENDPOINT) {
+      program.error('SOLANA_ENDPOINT environment variable not set')
+    }
+
+    const connection = new Connection(process.env.SOLANA_ENDPOINT)
+    const feePayer = Keypair.fromSecretKey(
+      Uint8Array.from(JSON.parse(process.env.SOLANA_FEEPAYER_SECRET_KEY))
+    )
+
+    const paymentRouterPublicKey = new PublicKey(
+      process.env.SOLANA_PAYMENT_ROUTER_PUBLIC_KEY
+    )
+
+    const tokenMint =
+      mint === 'audio'
+        ? new PublicKey(process.env.SOLANA_TOKEN_MINT_PUBLIC_KEY)
+        : new PublicKey(process.env.SOLANA_USDC_TOKEN_MINT_PUBLIC_KEY)
+
+    const senderAccountKeypair = Keypair.fromSecretKey(
+      Uint8Array.from(JSON.parse(process.env.SOLANA_OWNER_SECRET_KEY))
+    )
+    const senderAccountPublicKey = senderAccountKeypair.publicKey
+
+    try {
+      console.log('checking for source usdc account')
+      const senderTokenAccount =
+        await solanaWeb3Manager.findAssociatedTokenAddress(
+          senderAccountPublicKey.toString(),
+          mint
+        )
+      let senderTokenAccountInfo = await solanaWeb3Manager.getTokenAccountInfo(
+        senderTokenAccount.toString()
+      )
+
+      // If it's not a valid token account, we need to make one first
+      if (!senderTokenAccountInfo) {
+        console.log(
+          'Provided recipient solana address has no associated token account, creating'
+        )
+        const { blockhash, lastValidBlockHeight } =
+          await connection.getLatestBlockhash()
+        const accountCreationTx = new Transaction({
+          blockhash,
+          lastValidBlockHeight
+        })
+        const createTokenAccountInstruction =
+          createAssociatedTokenAccountInstruction(
+            senderAccountPublicKey, // fee payer
+            senderTokenAccount, // account to create
+            senderAccountPublicKey, // owner
+            tokenMint // mint
+          )
+        accountCreationTx.add(createTokenAccountInstruction)
+
+        const accountCreationTxSignature = await sendAndConfirmTransaction(
+          connection,
+          accountCreationTx,
+          [senderAccountKeypair],
+          {
+            skipPreflight: true
+          }
+        )
+        console.log(chalk.green(`Successfully created new ${mint} account`))
```

Comment:
ditto (nit) - say "token account" instead of just "account"

Reply:
Yep also done